packages/graphql/src/translate/batch-create/types.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type GraphQLCreateInput = Record<string, any>;
+
+export interface TreeDescriptor {
+    properties: Set<string>;
+    childrens: Record<string, TreeDescriptor>;
```

Comment:
```suggestion
    children: Record<string, TreeDescriptor>;
```

Reply:
I had to google it, to understand this correction, thank you!